### ISSUE TITLE
Option to set country

### ISSUE THREAD

**Issue by tindzk** [dirkgroenen/mopidy-mopify#35]
I came across the following code fragment and I'm wondering whether an option to configure the user's country could be added:

```javascript
Spotify.search($scope.searchQuery, searchableItems, {
    market: "NL",
    limit: "3"
})
```

(Source: https://github.com/dirkgroenen/mopidy-mopify/blob/master/src/app/music/stations/stations.controller.js#L76)

**Comment by dirkgroenen** [comment]
Ah forgot to change that when launching the first version. As Spotify mentions in their documentation this can be a alpha-2 country code or token. 
```An ISO 3166-1 alpha-2 country code or the string from_token. ```

**Comment by dirkgroenen** [comment]
Fixed in 1.1.5
